src/s1/t5/n1/exercici3/Main.java

```diff
@@ -0,0 +1,18 @@
+package s1.t5.n1.exercici3;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        try (Scanner scanner = new Scanner(System.in)) {
```

Comment:
//You can reduce the code and ensure the scanner is closed properly after the try block

public class Main {
    public static void main(String[] args) {
        try (Scanner scanner = new Scanner(System.in)) {
            String dirPath = DirectoryListerToFile.promptForPath(scanner);
            String outputFilePath = DirectoryListerToFile.promptForPath(scanner);
            DirectoryListerToFile.listDirContentsRecursivelyToFile(dirPath, outputFilePath);
        } catch (IOException e) {
            System.out.println("Se produjo un error: " + e.getMessage());
        }
    }
}